ui/app/components/clients/page/token.hbs

```diff
@@ -96,7 +97,7 @@
       this.isCurrentMonth
       "Only totals are available when viewing the current month. To see a breakdown of new and total clients for this month, select the 'Current Billing Period' filter."
     }}"
-    @isSecretsSyncActivated={{@isSecretsSyncActivated}}
```

Comment:
This was a bug that I came across and decided to address in this PR. The token tab is just for entity and non-entity client data and so the usage chart here should never render any sync-related data. 

I updated the arg to the `UsageStat` component to make the distinction clearer, since the component doesn't care about the "activated" state, just which stats to show. 

Another arg will be added to hide ACME clients, or the logic here will change to accept an array of stats to hide/show. But that felt like a follow on task for when the overview page is updated to include ACME clients